### DISCUSSION TITLE
Stop button font from resizing when clicked

### DISCUSSION THREAD

**Discussion by Borderliner** [saadeghi/daisyui#3014] [General]
How do I stop a button text from changing sizes when being clicked? I want the font size to stay the same and not get smaller.

**Comment by saadeghi** [comment]
https://daisyui.com/components/button/#button-without-click-animation

**Comment by Borderliner** [comment]
@saadeghi No not the whole animation. I the text to stay put, while the button shape resizes. Is that possible?

**Reply by saadeghi** [reply]
The text is in the button. How is the button supposed to resize without the text? Even if it was possible it would be a weird animation.

**Reply by Borderliner** [reply]
I understand. Thank you.
